src/core/Notification/BaseNotification/BaseNotification.baseStyles.ts

```diff
@@ -0,0 +1,73 @@
+import { css } from 'styled-components';
+import { SuomifiTheme } from '../../theme';
+import { element, font } from '../../theme/reset';
+
+export const baseNotificationBaseStyles = (theme: SuomifiTheme) => css`
+  &.fi-notification {
+    ${element(theme)}
+    ${font(theme)('bodyTextSmall')}
+    p {
+      font-size: 16px;
+    }
+    & .fi-notification_label {
+      ${font(theme)('bodySemiBold')}
+      margin-top: -4px;
+      margin-bottom: ${theme.spacing.xxs};
+    }
+
+    padding-bottom: 17px;
+    width: 100%;
```

Comment:
All the same level rules should be at the same position, now it seems that they are scattered all over the file.
_Looking  `box-shadow` and `border-radius` rules that are introduced later in the file_

Reply:
And these rules at this level should be in the top section before any others, e.g

```tsx
export const baseNotificationBaseStyles = (theme: SuomifiTheme) => css`
  &.fi-notification {
    ${element(theme)}
    ${font(theme)('bodyTextSmall')}

//  --> this place should be more appropriate for this level of styles <--

    p {
      font-size: 16px;
    }
    & .fi-notification_label {
      ${font(theme)('bodySemiBold')}
      margin-top: -4px;
// ... and the rest of styles file
```

Reply:
fixed